GameLogic/player_management/User.cs

```diff
@@ -21,7 +26,19 @@ public Player(string username)
         Gold = 0;
         Energy = 20;
         // Inventory = new List<Item>();
+        Equiped = new Equipment();
+    }
+    
+    
+    
+    public int TotalDefence()
+    {
+        if (Equiped == null)
+            return 0;
+        return Equiped.PlayerArmour.TotalDefence();
```

Comment:
What for? And why in this class? Why can't this be stuffed into a classroom with equipment?